doc/source/how-to/setup.rst

```diff
@@ -0,0 +1,451 @@
+
+            <venv>\Scripts\Activate.ps1
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            source <venv>/bin/activate
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            source <venv>/bin/activate
+
+Deactivate
+~~~~~~~~~~
+
+.. tabs:: 
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            deactivate
+
+
+Git
+---
+.. raw:: html
+    
+    <div align="center">
+      <img src="https://github.com/git/git-scm.com/raw/main/public/images/logo%402x.png">
+      <br><br>
+    </div>
+
+`Git <https://git-scm.com/>`_ is an open-source version control system (VCS). It
+is used to track changes and register new ones in software related projects. Git
+will register author and dates too, so accurate tracking of the evolution of the
+software can be made.
+
+Installation
+~~~~~~~~~~~~
+
+.. tabs::
+
+    .. group-tab:: Windows
+
+        To install Python in a machine using Windows, follow these steps:
+        
+        1. Download the `latest stable standalone Git version for Windows <https://www.python.org/downloads/win/>`_.
+        2. Execute the installer and follow the different steps.
+
+    .. group-tab:: macOS
+
+        To install Python in a machine using macOS, follow these steps:
+        
+        1. Check the `latest stable Git version for macOS <https://git-scm.com/download/mac>`_.
+        2. Run the installation command for your package manager.
+
+    .. group-tab:: Linux/UNIX
+
+        To install Git in a machine using Linux/UNIX, follow these steps:
+        
+        1. Check the `latest stable Git version for Linux/UNIX <https://git-scm.com/download/linux>`_.
+        2. Run the installation command for your package manager.
+
+
+Verify Installation
+~~~~~~~~~~~~~~~~~~~
+Check if the installation process succeeded by running the following command:
+
+.. tabs:: 
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            git --version
+
+Usage
+~~~~~
+If you are not familiar with this tool and would like to know how to use it,
+please refer to the `Git Reference Manual <https://git-scm.com/doc>`_. In
+previous website, you will find plenty of resources for learning Git.
+
+Configuration
+~~~~~~~~~~~~~
+It is very important to properly configure Git so every modification you made to
+the code properly points towards you. There are two types of configuration:
```

Comment:
```suggestion
It is very important to properly configure Git so that every modification that you make
to the code points to you. There are two types of configuration:
```